monitoring/grafana/grr_grafanalib_dashboards/config.py

```diff
@@ -0,0 +1,12 @@
+from grr_grafanalib_dashboards import reusable_panels
+
+# The data source name is specified after Grafana is set up
+# and it can be visited at localhost:3000.
+# In GRR Monitoring docs, we suggest naming it "grr-server", but if it's
```

Comment:
Nit: would it make sense to put a link to the docs you mention here?

Reply:
I will; just note that the documentation exists, but this bit is a part of a [PR](https://github.com/google/grr-doc/pull/117) to grr-doc which isn't merged at the moment.

Reply:
Ah, my bad. Merged.